ui/src/app/kvm/components/KVMResourcesCard/KVMResourcesCard.tsx

```diff
@@ -56,10 +57,40 @@ const KVMResourcesCard = ({
       <div className="kvm-resources-card__section kvm-resources-card__ram">
         <div>
           <h4 className="p-heading--small">RAM</h4>
-          <div className="kvm-resources-card__ram-chart">
-            {ram.general.total}
-            {ram.general.unit}
-          </div>
+          <DoughnutChart
+            label={`${ram.general.total + (ram.hugepage?.total || 0)}${
+              ram.general.unit
+            }`}
+            segmentHoverWidth={20}
```

Comment:
Think it's worth defining all these numbers as `const`s?

Reply:
Yeah probably, but might be worth waiting until this chart is used a second time and see if they stay the same.